spec/lib/oops_request/states_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe 'OopsRequest::States' do
+  let(:request) { create :oops_request }
+
+  it 'have state pending' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.